kafka-connect-s3/docs/s3_connector.rst

```diff
@@ -230,9 +230,9 @@ Finally, stop the Connect worker as well as all the rest of the Confluent servic
 
 Your output should resemble:
 
-.. include:: ../../../../includes/installation.rst
-    :start-line: 102
-    :end-line: 118
+.. include:: ../../../../includes/cli.rst
+    :start-line: 55
+    :end-line: 71
 
 or stop all the services and additionally wipe out any data generated during this quickstart by running:
```

Comment:
Would `Alternatively, stop all the services..` be a better way to transition to the next paragraph?

Reply:
yes, or possibly a tip that includes this information. however, this PR isn't meant to be a general cleanup of this doc - only fixing the single issue around single-sourcing the command output.